services-transport-rsocket/src/main/java/io/scalecube/services/transport/rsocket/ExtendedEpollEventLoopGroup.java

```diff
@@ -13,16 +13,18 @@
 import io.netty.util.concurrent.RejectedExecutionHandler;
 import io.netty.util.concurrent.RejectedExecutionHandlers;
 import java.lang.reflect.Constructor;
+import java.util.Iterator;
 import java.util.concurrent.Executor;
 import java.util.concurrent.ThreadFactory;
+import java.util.function.BiFunction;
 
 /**
  * Epoll event loop group with provided event executor chooser. Creates instances of {@link
  * io.netty.channel.epoll.EpollEventLoop}.
  */
 public class ExtendedEpollEventLoopGroup extends MultithreadEventLoopGroup {
 
-  private final EventExecutorChooser eventExecutorChooser;
+  private final BiFunction<Channel, Iterator<EventExecutor>, EventExecutor> eventExecutorChooser;
```

Comment:
Let's make it `eventLoopChooser` and set it like:
 `BiFunction<Channel, Iterator<EventExecutor>, EventLoop> eventLoopChooser`

Reply:
+